pkg/controller/nodes/subworkflow/launchplan/admin_test.go

```diff
@@ -193,6 +195,42 @@ func TestAdminLaunchPlanExecutor_Launch(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("happy recover", func(t *testing.T) {
```

Comment:
what if recover fails?

Reply:
updated to call create execution when applicable